### DISCUSSION TITLE
Multilabel training and detection or not?

### DISCUSSION THREAD

**Discussion by lkuebler** [ultralytics/yolov5#7673] [Q&A]
Hello Yolov5 Team

I've been using and deploying your model since a few months. It is amazing!

I have multiple labels for the same bounding boxes in my dataset. Until now I just duplicated the bounding boxes in their respective txt files. Is there a way to train with multiple labels per bounding box?
I am asking this because when detecting using either detect.py or Pytorch Hub with agnostic set to False, I sometimes only get one bounding box  with one label instead of two boxes with different labels. 
If I set agnostic to True do I get multiple labels per bounding box? How do you extract them with the result from the Pytorch Hub model?

**Comment by glenn-jocher** [comment]
@lkuebler there are some multi-label training PRs open here, but we have not followed up on them as we haven't had time to test sufficiently:

https://github.com/ultralytics/yolov5/pull/5593
